script.module.resolveurl/lib/resolveurl/plugins/cloudb.py

```diff
@@ -10,20 +10,20 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Do not remove these spaces. This is as per English grammar, two spaces after period, single space after comma

Reply:
What? There is one space after a period.

Reply:
Copyleft is a legal document in Monospace type font, so there must be two spaces after period.

Reply:
Now aren't we just referencing the gpl in the doc string?
Plus this `https://www.gnu.org/licenses/old-licenses/gpl-2.0.html` html version of the license does not include 2 spaces after a period.

Reply:
Also found some examples they provide for license notices to include like the one in our doc string. `https://www.gnu.org/licenses/gpl-howto.html` and which they also have the normal one space. Cant find anything that states otherwise.

My main reason for  removing the unnecessary space (besides the fact that nobody types like that) is to clean things up. Just like trying to be consistent with string quotes. Makes things easier on the eyes when needing to read/work on code.

Reply:
I totally agree with replacing double quotes with single for strings.
The spacing rules as I said above, is to use two spaces when using monospaced type font and a single space when you use Proportional type font. This is a typography standard. You always look at Source code in monospaced font and so the double space requirement.
The HTML examples that you mention above are using proportional font and hence would use single space

Reply:
While i do personally prefer single over double, the goal is just to get them consistent per py file vs a weird mix of both as there is no reason for that madness.

We also don't code on typewriters, but will try to adhere to this moving forward if you REALLY prefer 2 spaces in the doc string. Your project bud 😄

Reply:
Okay all files doc strings made consistent (2 spaces), all string with double quotes changed to single.
Please continue contributing with current format for any new PRs, I have also made some changes to generic resolvers for better readability of long lines, for e.g., [this line](https://github.com/Gujal00/ResolveURL/blob/4dbe0ee8427e76a9b65a550f2ab6009f4a7a4d23/script.module.resolveurl/lib/resolveurl/plugins/avideo.py#L29)
As always, Thanks for the contributions. Cheers!